routers/api/v1/repo/board.go

```diff
@@ -0,0 +1,232 @@
+// Copyright 2016 The Gogs Authors. All rights reserved.
+// Copyright 2018 The Gitea Authors. All rights reserved.
```

Comment:
Since this is a new file, it just needs Gitea Authors copyright and the year should be 2022